Developer-Guide.md

```diff
@@ -186,6 +186,13 @@ $ sudo docker run -ti --runtime kata-runtime busybox sh
 
 # Troubleshoot
```

Comment:
That wording was recommended by @klynnrif so I'll let you guys fight it out ;)

Reply:
@grahamwhaley @jodh-intel I would agree in general, but the style guidelines I originally saw mentioned no use of the Gerund (i.e. the "ing" form). That being said, I would defer to @rcaballeromx on this one. :)

Reply:
@klynnrif is correct. Please avoid gerunds because they lower SEO. However, it is important to mention what is being trobleshooted. I suggest: Troubleshoot Kata Containers